Resources/Help/preferences/PrefsResults.md

```diff
@@ -11,9 +11,9 @@ Regarding analysis results in JASP you can choose the following options:
 
 Here you can specify if p-values should be shown as "< .001" or if an exact value should be given.
 
-### Use normalized notation
+### Use exponent notation
 
-Here you can specify if scientific numbers should be displayed with normalized notation, 1×10<sup>-10</sup> instead of being displayed using the E notation, 1e-10.
+Here you can specify if scientific numbers should be displayed with exponent notation, 1×10<sup>-10</sup> instead of being displayed using the E notation, 1e-10.
```

Comment:
Now it says "displayed with exponent ... instead of ... using the E notation" which is contradictory

Reply:
Ive changed it